sideline-models/sideline_interface.go

```diff
@@ -0,0 +1,7 @@
+package sideline_models
+
+type CheckMessageSideline interface {
```

Comment:
we don't need to define interface explicitly, we can keep sideline related code under single file.